packages/template-retail-react-app/app/pages/home/index.jsx

```diff
@@ -73,6 +74,14 @@ const Home = () => {
         einstein.sendViewPage(pathname)
     }, [])
 
+    //TODO: Remove after PR is approved. This is for hook testing and verifying
+    const {data} = useUrlMapping({
+        parameters: {
+            urlSegment: '/sample-product'
+        }
+    })
+    console.log('data', data)
```

Comment:
After this test code is removed, will the retail-react-app show an example of how to really use `useUrlMapping`?

For example, right now I don't know whether I'll need to update my routes. Where should I call `useUrlMapping`? 

I think it would be good if we can leave commented code somewhere, as a best practice on how to use this url mapping feature.

Reply:
@vmarta I think useUrlMapping is like any other query hooks in the library.  You can certainly call it any where you want in the app as long as you know which parameters to pass in the hook. (we currently do not have any examples for unused hooks in commerce-react)
In addition, we will start the Support ShopperSEO url mapping [work](https://salesforce.quip.com/92mOAfJ4uZc2) soon, I think we will have more context to document there.